.github/ci_configs/vllm/Dockerfile.tmp_vllm

```diff
@@ -338,20 +338,23 @@ ARG torch_cuda_arch_list='8.0;8.9;9.0a;10.0a;12.0'
 # install package for build flashinfer
 # see issue: https://github.com/flashinfer-ai/flashinfer/issues/738
 
-RUN pip install build==1.3.0
-RUN pip freeze | grep -E 'setuptools|packaging|build'
+RUN --mount=type=cache,target=/root/.cache/uv \
+    uv pip install  --system build==1.3.0
+RUN uv pip freeze | grep -E 'setuptools|packaging|build'
 
 ENV TORCH_CUDA_ARCH_LIST=${torch_cuda_arch_list}
+ENV FLASHINFER_CUDA_ARCH_LIST=${torch_cuda_arch_list}
```

Comment:
Btw, there is a new script now to build FlashInfer from vLLM https://github.com/vllm-project/vllm/pull/25782 called `tools/flashinfer-build.sh`.  You could try to see if that script works for us here (later)